freedom/likelihood.py

```diff
@@ -53,7 +53,10 @@ def __call__(self, event, params):
         n_points = params.shape[0]    
 
         # Charge Net
-        inputs = [np.repeat(event['total_charge'], repeats=n_points)[:, np.newaxis], params]
+        if type(event['total_charge']) == np.float:
+            inputs = [np.repeat(event['total_charge'], repeats=n_points)[:, np.newaxis], params]
+        else:
+            inputs = [np.array([event['total_charge'],]*n_points), params]
```

Comment:
Why is this differentiation of cases? Also don't check for dtype like that?
Also, list -> array is a bad idea....why not use np.repeat?